guides/doc-Upgrading_and_Updating/topics/con_following-the-progress-of-the-upgrade.adoc

```diff
@@ -0,0 +1,8 @@
+[[following_the_progress_of_the_upgrade]]
+== Following the Progress of the Upgrade
+
+Because of the lengthy upgrade time, use a utility such as `tmux` to suspend and reattach a communication session.
```

Comment:
I don't think suspend is a good description, rather detach?
```suggestion
Because of the lengthy upgrade time, use a utility such as `tmux` to detach and reattach a communication session.
```

Reply:
@ekohl what if we use 'suspend and resume' instead?